### ISSUE TITLE
Random crash after some playing time

### ISSUE THREAD

**Issue by dbvasconcelos** [ytmdesktop/ytmdesktop#295]
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**.
- [X] I verified that there is no open issue for the same subject.

**Describe the bug**
The app crashes randomly after some time playing on background 

**To Reproduce**
1. Start the app
2. Hit play
3. Close the window to let it run on background
4. Wait a few minutes
4. App crash without any message

**Expected behavior**
To not crash on background

**Screenshots**
since it crashes without any popup, all i have is the systemd log:
```
ago 06 18:44:43 stark systemd[1]: Started Process Core Dump (PID 213265/UID 0).
ago 06 18:44:43 stark audit[1]: SERVICE_START pid=1 uid=0 auid=4294967295 ses=4294967295 subj==unconfined msg='unit=systemd-cored>
ago 06 18:44:43 stark kernel: audit: type=1130 audit(1596750283.670:179): pid=1 uid=0 auid=4294967295 ses=4294967295 subj==unconf>
ago 06 18:44:44 stark systemd-coredump[213266]: Process 205988 (youtube-music-d) of user 1000 dumped core.

                                                Stack trace of thread 205988:
                                                #0  0x00007f5e3d070621 n/a (libappindicator3.so.1 + 0x7621)
                                                #1  0x00007f5e59fb910d n/a (/snap/youtube-music-desktop-app/4/gnome-platform/usr/>
-- Process 205988 (youtube-music-d) crashed and dumped core.
--
-- This usually indicates a programming error in the crashing program and
-- should be reported to its vendor as a bug.
```

**Environment (please complete the following information):**
 * YTMDesktop version: ***1.12.0***
 * OS: ***Arch Linux***
 * OS version: ***5.7.12-arch1-1***
 * Arch: ***x64***
 * WM: ***dwm***
 * Installation way: ***snapcraft***

**Comment by dbvasconcelos** [comment]
Just realized it happens when the window is active as well.

**Comment by adlerluiz** [comment]
@rickinbsb

**Comment by dbvasconcelos** [comment]
Just giving the feedback that apparently this issue was solved on the 1.12.1 version.

**Comment by rickinbsb** [comment]
Amazing! I'll close this, but fell free to reopen it in case this happens again
